lxd/metrics/api.go

```diff
@@ -60,6 +60,7 @@ type MemoryMetrics struct {
 	SwapBytes           uint64 `json:"memory_swap_bytes" yaml:"memory_swap_bytes"`
 	UnevictableBytes    uint64 `json:"memory_unevictable_bytes" yaml:"memory_unevictable_bytes"`
 	WritebackBytes      uint64 `json:"memory_writeback_bytes" yaml:"memory_writeback_bytes"`
+	OOMKills            uint64 `json:"memory_oom_kills" yaml:"memory_oom_kills"`
```

Comment:
@simondeziel in Prometheus do we have to indicate if a variable is a gauge or a counter?

Reply:
In this case its a counter.

Reply:
Otherwise if your graphing this metric and the instance restarts the counter will reset and will skew the rate calculation right?

Reply:
Ideally, the metric label would have the `_total` suffix so `memory_oom_kills_total` which implies it's a counter.

Reply:
> In this case its a counter I think.

Yes, it is counter.

Reply:
> Otherwise if your graphing this metric and the instance restarts the counter will reset and will skew the rate calculation right?

That's the nice thing about counters, the Prometheus functions we use (`rate()` and such) detect resets and cope with them as if the counter kept increasing.

Reply:
> Otherwise if your graphing this metric and the instance restarts the counter will reset and will skew the rate calculation right?

yes

Reply:
How does it know its a counter rather than a gauge?

Reply:
It doesn't know (or really need to), it's just that some functions are meant to work on counters, not gauges.

Reply:
Got you.